src/pages/index.js

```diff
@@ -164,31 +168,36 @@ editAvatarValidator.enableValidation();
 /*                      Functions                   */
 /*--------------------------------------------------*/
 
-function handleDeleteCardClick() {
+function handleDeleteCardClick(card) {
   confirmDeleteModal.open();
   confirmDeleteModal.setSubmitAction(() => {
-    const id = this.getId();
-
+    const id = card.getId();
     api
       .removeCard(id)
       .then((res) => {
-        this.handleDeleteCard();
+        card.handleDeleteCard();
       })
       .catch((err) => {
         console.error(err);
       });
   });
 }
 
-function handleCardLikeClick(cardId, isLiked) {
-  if (isLiked) {
+function handleCardLikeClick(card) {
+  if (card.getIsLiked()) {
     api
-      .removeCardLike(cardId)
-      .then((cardData) => this.setLike(cardData.isLiked));
+      .removeCardLike(card.getId())
+
+      .then((cardData) => card.setLike(cardData.isLiked));
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**